README.md

```diff
@@ -43,433 +44,67 @@ Thank you for taking the time to participate in our last survey. You can continu
 
  Please visit the [blog][blog] for more announcements.
 
+## Prerequisites
 
-## Build (Windows)
-
-Note: if you prefer, you can use the pre-compiled binaries found in the [releases](https://github.com/Microsoft/msphpsql/releases)
-
-#### Prerequisites
-
-You must first be able to build PHP 7.0.* or above without including these extensions.  For help with doing this, see the [official PHP website][phpbuild] for building your own PHP in Windows.
-
-#### Compile the drivers
-
-The Microsoft Drivers for PHP for SQL Server have been compiled and tested with PHP 7.0.* and 7.1.* using Visual C++ 2015 as well as PHP 7.2.0* using Visual C++ 2017 v15.0. 
-For details, please read the documentation and/or take a look at the sample [build scripts](https://github.com/Microsoft/msphpsql/tree/dev/buildscripts#windows).
-
-## Install (Windows)
-
-#### Prerequisites
+For full details on the requirements for the drivers, see the [system requirements](https://docs.microsoft.com/en-us/sql/connect/php/system-requirements-for-the-php-sql-driver) on MSDN.
 
+On the client machine:
+- PHP 7.0.x, 7.1.x, or 7.2.1 and up on Windows, or 7.2.0 and up on Unix systems 
 - A Web server such as Internet Information Services (IIS) is required. Your Web server must be configured to run PHP
```

Comment:
please rephrase your sentence, or else it reads like we don't support 7.0* or 7.1* in Unix